src/cypress/support/commands.js

```diff
@@ -0,0 +1,220 @@
+})
+
+Cypress.Commands.add('clickInitateRFQ', symbol => {
+  tradePage.getRFQButton(symbol.replace('/', '')).click()
+})
+
+Cypress.Commands.add('verifyEnabledDirections', symbol => {
+  tradePage.getDirectionButton(symbol.replace('/', ''), 'Buy').should('not.be.disabled')
+  tradePage.getDirectionButton(symbol.replace('/', ''), 'Sell').should('not.be.disabled')
+})
+
+Cypress.Commands.add('verifyRequoteButton', symbol => {
+  tradePage.getRFQButton(symbol.replace('/', '')).then(el => {
+    expect(el.text()).to.be.equal('Requote')
+  })
+})
+
+Cypress.Commands.add('editNotionalValue', (notional, symbol) => {
+  cy.reload()
+  tradePage
+    .enterNotional(symbol.replace('/', ''))
+    .clear()
+    .type(notional)
+})
+
+Cypress.Commands.add('clickOnRejectButton', symbol => {
+  tradePage.getRejectButton(symbol.replace('/', '')).click()
+})
+
+Cypress.Commands.add('clickOnRejectButton', symbol => {
+  tradePage.getRejectButton(symbol.replace('/', '')).click()
+})
+
+Cypress.Commands.add('clickOnRequoteButton', symbol => {
+  tradePage.getRequoteButton(symbol.replace('/', '')).click()
+})
+
+Cypress.Commands.add('clickOnRequoteButton', symbol => {
+  tradePage.getRequoteButton(symbol.replace('/', '')).click()
+})
+
+Cypress.Commands.add('verifyNotionalValue', (expected, symbol) => {
+  tradePage.getNotionalValue(symbol.replace('/', '')).should('contain', expected)
+})
+
+Cypress.Commands.add('verifyErrorMessage', (expected, symbol) => {
+  cy.contains(expected).should('be.visible')
+})
+
+Cypress.Commands.add('performTrade', (symbol, notional, direction) => {
+  currency = symbol.replace('/', '')
+
+  cy.log('').then(() => {
+    tradePage
+      .enterNotional(currency)
+      .clear()
+      .type(notional)
+    tradePage.getCurrentPrice(currency, direction).each((el, index, list) => {
+      if (index == 0) {
+        // to get the current price text from the selected element
+        currentPrice = el.find('div:nth-child(2)').text()
+      } else {
+        currentPrice = currentPrice + el.text()
+      }
+    })
+  })
+
+  cy.log('').then(() => {
+    cy.log('current price is ' + currentPrice)
+    if (Number(currentPrice) % 10 == 0) {
+      currentPrice = Number(currentPrice) / 10
+    }
+    finalAmount = Number(currentPrice) * Number(notional)
+
+    tradePage.getDirectionButton(currency, direction).click({ force: true })
+    if (currency == 'EURJPY') {
+      cy.wait(2000)
+      tradePage.getWaitingTextMessage(currency).then(el => {
+        cy.log('waiting text ' + el.text())
+        waitingText = el.text()
+
+        cy.wait(3000)
+      })
+    } else {
+      cy.wait(5000)
+    }
+
+    tradePage.getConfirmationTextMessage(currency).then(el => {
+      cy.log('Confirmation text ' + el.text())
+      confirmationText = el.text()
+    })
+  })
+})
+
+Cypress.Commands.add('verifyTradeRejected', () => {
+  cy.log('').then(() => {
+    expect(confirmationText).to.eq('Your trade has been rejected')
```

Comment:
This assertion is better, but see earlier comment about using state (confirmationText) inside the commands file.
Also, with an assertion like this, we shouldn't need to do it inside a `cy.log()`?